README.md

```diff
@@ -13,6 +13,7 @@ This repo can be installed on either the same VM as the main LORIS codebase, or
 On <u>Ubuntu</u>, DICOM toolkit will be installed by the imaging install script (step 4 below). This script will _apt-get install dcmtk_.   
```

Comment:
@christinerogers 
I only meant in the listing (the 3 bullets you have), just list MINC toolkit before DICOM toolkit; i.e. swapping lines 10 and 11, that's all.